src/main/java/seedu/address/MainApp.java

```diff
@@ -248,6 +255,7 @@ public void stop() {
         logger.info("============================ [ Stopping Address Book ] =============================");
         try {
             storage.saveUserPrefs(model.getUserPrefs());
+            storage.savePropertyBook(model.getPropertyBook());
```

Comment:
i dont think this is where you save the property book

Reply:
cause it only gets saved on exit but rmb for things like ip how they want to make sure if your application gets suddenly closed it will save till the last previous excuted command. I think me and Ms save it in one of the parser or managers

Reply:
Thanks Marucs. Yes this part is redundant, will delete it.